### ISSUE TITLE
NASCAR THE GAME 2011 [BLUS30604] - no visual damages on cars

### ISSUE THREAD

**Issue by segbed** [RPCS3/rpcs3#15667]
### Quick summary

Independently on in-games damage setting cars have no any visual damages when crashing, hiting, etc.
I sow some confirmation comments under youtube videos.
99.9% sure it is issue relevant independently on PC configuration and RPCS3 settings.
I've tried a lot to fix it by changing RPCS3 settings and dealing woth this issue already 2-3 years.
PS it is my favorite nascar game)

### Details

On real PS3 for sure car damage model works fine. It is enough to touch wall on low speed to see scratches and deformations on car body.
https://youtu.be/tN8z03J9LwU?si=mS0kjXErv7-5ao2N&t=56

On RPCS3 you can touch a wall on 200mph speed with no visual damages on car body.

### Attach a log file

[n2011_RPCS3.log.gz](https://github.com/user-attachments/files/15653843/n2011_RPCS3.log.gz)

### System configuration

b650
7800x3d
RTX 4090
32GB DDR5 6000
Windows 11

**Comment by SanokMaynik** [comment]
I have the same issue but only sometimes, randomly the damage just works again without me doing anything

**Comment by segbed** [comment]
yes, I can confirm this behavior, it can work just randomly for one game run. If it run ant damages work - it work. Otherwise it is not. Something randomly happening during the start, possibly some timings, delays, etc. It also seems not related to host platform/performance sine I've completely upgraded my rack from 10900 to 7800x3d to 9800x3d, no changes. Hopefully sometime it will be fixed, but since there are not many fans of this game we possibly will need to wait some time.

**Comment by CT-6924** [comment]
YES YES YES I wish I could figure out why it doesn't work. I hope this post gains some traction.

**Comment by segbed** [comment]
There is no progress at the moment, I would even say that the situation has worsened, since I have not been able to launch this game with working damage visual effects for the last 5-6 times.

**Comment by digant73** [comment]
surely still an issue but please provide an update on this issue
